slices/libgdk-pixbuf-2.0-0.yaml

```diff
@@ -0,0 +1,28 @@
+package: libgdk-pixbuf-2.0-0
+
+essential:
+  - libgdk-pixbuf-2.0-0_copyright
+
+slices:
+  bins:
+    essential:
+      - libgdk-pixbuf-2.0-0_libs
+    contents:
+      /usr/lib/*-linux-*/gdk-pixbuf-2.0/gdk-pixbuf-query-loaders:
+
+  libs:
+    essential:
+      - libc6_libs
+      - libgdk-pixbuf2.0-common_copyright
```

Comment:
One final comment from my side: this feels a bit weird here. Why only `libs` need that package's copyright? I get you're assuming that this copyright will always be there because `bins` depends on `libs` (and thus will carry the copyright anyway). But that leaves from for future slices to "forget" this copyright dependency.

What about adding it as a libgdk-pixbuf-2.0-0_copyright dependency?

Reply:
Ahh this was an oversight from way back when, before I knew about (or before we had? IDK) a top-level `essential`.

Reply:
Done!